resources/light/samVisualize/primaryResources.json

```diff
@@ -0,0 +1,16 @@
+[
+    "AWS::CodeBuild::Project",
```

Comment:
this file should be called out in our developer docs somewhere. can you add a `## samVisualize` section in CONTRIBUTING.md after this section: https://github.com/aws/aws-toolkit-vscode/blob/e8af78cc02735411718ef982f364c1df3c473228/CONTRIBUTING.md#importing-icons-from-other-open-source-repos

to list key areas of knowledge needed for samVisualize maintenance